Content.Server/Chemistry/Components/ChemMasterComponent.cs

```diff
@@ -281,6 +281,9 @@ private void TryCreatePackage(EntityUid user, UiAction action, string label, int
             var handSys = _sysMan.GetEntitySystem<SharedHandsSystem>();
             var solSys = _sysMan.GetEntitySystem<SolutionContainerSystem>();
 
+            pillAmount = Math.Clamp(pillAmount, 0, 10);
```

Comment:
This should be a static const on some system because someone is going to update the spinbox and forget these.

Reply:
3913e5d87ca3fba8029e4269ade87d50fab3842d